{"cveId": "CVE-2022-32252", "cwe": ["CWE-345", "CWE-345", "CWE-345", "CWE-345", "CWE-345"], "cvss": [{"baseScore": "6.5", "version": "3.1", "Attack Vector": "Low", "Attack Complexity": "Low", "Privileges Required": "High", "User Interaction": "Required", "Scope": "Unchaged", "Confidentiality Impact": "High", "Integrity Impact": "High", "Availability Impact": "High"}], "references": ["https://cert-portal.siemens.com/productcert/pdf/ssa-484086.pdf", "https://cert-portal.siemens.com/productcert/html/ssa-484086.html"], "description": ["A vulnerability has been identified in SINEMA Remote Connect Server (All versions < V3.1). The application does not perform the integrity check of the update packages. Without validation, an admin user might be tricked to install a malicious package, granting root privileges to an attacker."], "published": "2022-06-14T10:15:20.763", "state": "PUBLIC", "vendorName": ["Siemens"], "productName": ["SINEMA Remote Connect Server"], "github": {"advisories": [], "commits": [], "pocAdvisorie": null, "repo": null, "info": {}}, "pocList": []}